### ISSUE TITLE
Grep version from busybox issue with OhMyZsh grep-alias cache

### ISSUE THREAD

**Issue by hansemschnokeloch** [loophp/nix-shell#175]
Hi, since d2940b4519ebb40e257d5fd0b4a909d2af4ba681 I've an issue with grep and ohmyzsh.

```
grep --version
grep: unrecognized option: exclude-dir=.bzr
BusyBox v1.36.1 () multi-call binary.
```

I think that this issue is due to the new use of minimal-nix-shell, and the caching mecanism used by ohmyszsh https://github.com/ohmyzsh/ohmyzsh/blob/fd219a94ab585fa699a0e842335a9f33dcbb613b/lib/grep.zsh#L4

Busybox grep version has to be used with --exclude option, and nixos grep with --exclude-dir

If the cache file exists and is not older than a day, the right option to be used with grep will not be selected https://github.com/ohmyzsh/ohmyzsh/blob/fd219a94ab585fa699a0e842335a9f33dcbb613b/lib/grep.zsh#L18-L22

**Comment by drupol** [comment]
Hi @hansemschnokeloch !

Thanks for using my PHP flake!

There have been a lot of changes lately, and I had in my to-do list to remove `mkshell-minimal`.

I've just published a PR that should fix your issue, once the CI is green, I'll merge it.

I let you close this PR when the issue is fixed for you.


Have a great Sunday.

**Comment by hansemschnokeloch** [comment]
Thanks @drupol !

Issue fixed with 1f428cd822923d4fbe7e99a62aba53e09a55db74

**Comment by drupol** [comment]
You're very welcome! Consider sponsoring if this is helping you, it would be greatly appreciated.
